daml-assistant/daml-helper/test/DA/Daml/Helper/Test/Deployment.hs

```diff
@@ -41,33 +41,42 @@ main = do
   damlHelper <- locateRunfiles (mainWorkspace </> "daml-assistant" </> "daml-helper" </> exe "daml-helper")
   let tools = Tools {..}
   defaultMain $ testGroup "Deployment"
-    [ authenticatedUploadTest tools
+    [ authenticationTests tools
     , fetchTest tools
     ]
 
--- | Test `daml ledger upload-dar --access-token-file`
-authenticatedUploadTest :: Tools -> TestTree
-authenticatedUploadTest Tools{..} = do
-  let sharedSecret = "TheSharedSecret"
+-- | Test `daml ledger list-parties --access-token-file`
+authenticationTests :: Tools -> TestTree
+authenticationTests Tools{..} =
   withSandbox defaultSandboxConf { mbSharedSecret = Just sharedSecret } $ \getSandboxPort ->
-    testCaseSteps "authenticatedUploadTest" $ \step -> do
-    port <- getSandboxPort
-    withTempDir $ \deployDir -> do
-      withCurrentDirectory deployDir $ do
-        writeMinimalProject
-        step "build"
-        callProcessSilent damlc ["build"]
-        let dar = ".daml/dist/proj1-0.0.1.dar"
-        let tokenFile = deployDir </> "secretToken.jwt"
-        step "upload"
-        -- The trailing newline is not required but we want to test that it is supported.
-        writeFileUTF8 tokenFile ("Bearer " <> makeSignedJwt sharedSecret <> "\n")
-        callProcessSilent damlHelper
-          [ "ledger", "upload-dar"
-          , "--access-token-file", tokenFile
-          , "--host", "localhost", "--port", show port
-          , dar
-          ]
+    testGroup "authentication"
```

Comment:
Isn't this just an instance of the compatibility test above where SDK and Ledger both use version `0.0.0`?

Reply:
Yes, we keep this for now since they are more convenient for development but I expect that we’ll probably kill them in the foreseeable future.